spec/controllers/authorizations_controller_spec.rb

```diff
@@ -985,7 +985,7 @@ def query_params
       end
 
       it "renders bad request" do
-        expect(response).to have_http_status(:bad_request)
+        expect(response).to have_http_status(:unauthorized)
```

Comment:
So request here is correct and user was just not authorized , right? Should we also change test name to reflect the check?

Reply:
Yep, done! Also added the changelog entry + added the config option w/ description to the initializer template